python/ml4ir/applications/classification/tests/generate_test_data.py

```diff
@@ -30,6 +30,7 @@
 ]
 VOCABULARY_FEATURE_DOMAIN_ID = [char for char in string.ascii_lowercase.upper()]
 VOCABULARY_FEATURE_ENTITY = ["AAA", "BBB", "CCC", "DDD", "EEE", "FFF", "GGG", "HHH"]
+VOCABULARY_FEATURE_CODE = ["local", "production", "canary"]
```

Comment:
Thanks for adding this, could we comment it
 VOCABULARY_FEATURE_CODE = ... # untrainable feature, used to compute grouped metric based on this category 
or something like that to provide context?